_content/articles/gradual-entry-into-emptiness_analayo.md

```diff
@@ -0,0 +1,23 @@
+---
+title: "Gradual entry into emptiness - depicted in the early Buddhist discourses"
+authors:
+  - analayo
+external_url: "https://www.buddhismuskunde.uni-hamburg.de/pdf/5-personen/analayo/gradual-emptiness.pdf"
+source_url: ""
+drive_links:
+  - "https://drive.google.com/file/d/12bhfpHGO0_Ai-vUCwpz4_oK89GvODFLO/view?usp=sharing"
+course: samadhi
+tags:
+  - ma
```

Comment:
Usually when there's a cluster of related tags like this I consider up-leveling to the umbrella tag. In this case, I'm thinking just `agama` here instead of the three nikaya tags should capture that this is a good example of the comparative approach.